README.md

```diff
@@ -1,11 +1,11 @@
 
-ECMAScript2015(ES6) is the latest version of the ECMAScript standard. 
+ECMAScript 2016 (ES7) is the latest version of the ECMAScript standard. 
```

Comment:
Check @gaearon 's comments in [here](https://github.com/cht8687/React-ES5-To-ES6-Checklist/issues/4), 

> ES7 are experimental language proposals that may or may not get into the language someday.'